### PR TITLE
Add a dropdown for country field in user-profile pages of both Console and MyAccount

### PR DESCRIPTION
**Pull request by DinikaSen**
## Purpose
This PR introduces a dropdown for the country input field in user profile edit pages in Console and My Account.
Resolves https://github.com/wso2/product-is/issues/11631

**NOTE** : Do not merge until relevant cypress tests are updated accordingly

image

### CODE REVIEW

**Review by ChamathNS — APPROVED** [review]

File: apps/myaccount/src/components/profile/profile.tsx [hunk 1]
```diff
@@ -183,6 +185,28 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): J
         }
     }, [profileSchema, profileDetails.profileInfo]);
 
+    /**
+     * This will load the countries to the dropdown.
+     */
+    useEffect(() => {
+        const countryCodesToSkip = ["AQ", "BQ", "CW", "GG", "IM", "JE", "BL", "MF", "SX", "SS"];
```
Comment (brionmario) [inline apps/myaccount/src/components/profile/profile.tsx:192]: Is this needed since we have the util method? @DinikaSen

Comment (DinikaSen) [reply apps/myaccount/src/components/profile/profile.tsx:192]: This is for my account. Since the util method is in console, had to add this separately.

Comment (jeradrutnam) [reply apps/myaccount/src/components/profile/profile.tsx:192]: Let's move it to common utils and use. Create an issue to track it.

Comment (DinikaSen) [reply apps/myaccount/src/components/profile/profile.tsx:192]: Created an issue to track moving this to common utils - https://github.com/wso2-enterprise/asgardeo-product/issues/3364

Comment (DinikaSen) [reply apps/myaccount/src/components/profile/profile.tsx]: Addressed with https://github.com/wso2/identity-apps/pull/1972/commits/3a82b794fd61d8510b4f545ce44805b09f5681d4

**Pull request merged**